documentation/manage/updating-your-extension.md

```diff
@@ -0,0 +1,216 @@
+
+<!-- Syntax Highlighting -->
+
+{% highlight json linenos %}
+
+{
+"addons": {
+"{abcd1234-1abc-1234-12ab-abcdef123456}": {
+"updates": [
+{ "version": "0.1",
+"update_link": "https://example.com/addon-0.1.xpi" },
+{ "version": "0.2",
+"update_link": "http://example.com/addon-0.2.xpi",
+"update_hash": "sha256:fe93c2156f05f20621df1723b0f39c8ab28cdbeec342efa95535d3abff932096" },
+{ "version": "0.3",
+"update_link": "https://example.com/addon-0.3.xpi",
+"applications": {
+"gecko": { "strict_min_version": "44" } } }
+]
+}
+}
+}
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+## Enabling updates to your extension
+
+If your extension is not hosted on AMO, you must specify the location of your update manifest in your extension. For extensions developed with WebExtension APIs, add the following to your manifest:
+
+<!-- Syntax Highlighting -->
+
+{% highlight json linenos %}
+
+"applications": {
+"gecko": {
+"update_url": "https://example.com/updates.json"
+}
+}
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+For XUL add-ons, add the following to the `<Description about="urn:mozilla:install-manifest">` element of your `install.rdf` file:
+
+<!-- Syntax Highlighting -->
+
+{% highlight XML linenos %}
+
+<em:updateURL>https://example.com/updates.json</em:updateURL>
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+## Manifest Structure
+
+The manifest is a JSON file, with a top-level object literal. This object may have the following properties:
+
+<!-- Table -->
+
+{% capture table %}
+
+| Property | Type     | Description                                                                                                                                                                                                       |
+| -------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `addons` | `object` | An object containing one entry for each add-on to be updated. For each such entry, the name of the property must be the add-on's UUID, and the value must be an [object describing the add-on]() and its updates. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
+Properties of the `addons` object must contain object literals, each describing an add-on to update. These objects may have the following properties:
+
+<!-- Table -->
+
+{% capture table %}
+
+| Property  | Type               | Description                                                                 |
+| --------- | ------------------ | --------------------------------------------------------------------------- |
+| `updates` | `Array` (Optional) | An array containing zero or more update description objects for the add-on. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
```

Comment:
Needs h3 for Update Objects with highlighted line for `addons[*].updates[*]` underneath it

Reply:
Right above the h3 line, add `<section id="addons-updates"></section>`

This will give you an anchor link to this section.